src/SecTester.Core/IConfigurationOptions.cs

```diff
@@ -0,0 +1,12 @@
+using System.Collections.Generic;
+using Microsoft.Extensions.Logging;
+
+namespace SecTester.Core;
+
+public interface IConfigurationOptions
+{
+  string Hostname { get; }
+  Credentials? Credentials{ get; }
```

Comment:
Please fix formatting issues (even GitHub knows about this 😄 )
<img width="742" alt="image" src="https://user-images.githubusercontent.com/38690835/207672781-21b1898a-ee7b-4ba9-a98a-fb221c93fa50.png">